src/notifications/generate.ts

```diff
@@ -213,6 +213,7 @@ export const generateNotificationMap: Record<
       .objectPost(ctx.post, ctx.source, ctx.sharedPost),
   squad_blocked: (builder, ctx: NotificationSourceContext) =>
     builder
+      .targetUrl(process.env.COMMENTS_PREFIX)
```

Comment:
It should be added after `referenceSource` because it also sets the target url. So adding it will be overridden later.

Reply:
Only targetSource will set it right?
As the issue was it being empty

Reply:
My bad!